frontend/packages/dev-console/integration-tests/support/step-definitions/common/common.ts

```diff
@@ -1,18 +1,17 @@
 import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps';
-import { perspective, projectNameSpace, navigateTo } from '../../pages/app';
-import { switchPerspective, devNavigationMenu } from '../../constants/global';
+import { perspective, projectNameSpace, navigateTo } from '../../pages';
+import { switchPerspective, devNavigationMenu, adminNavigationMenu } from '../../constants';
 import { guidedTour } from '../../../../../integration-tests-cypress/views/guided-tour';
 import { nav } from '../../../../../integration-tests-cypress/views/nav';
 import { modal } from '../../../../../integration-tests-cypress/views/modal';
 import { detailsPage } from '../../../../../integration-tests-cypress/views/details-page';
```

Comment:
add index.ts in /views and import this in one line.

Reply:
This is related to console folders, So i haven't touched them

Reply:
Oh okay, Please use `@console/cypress-integration-tests/` instead of relative imports.